### ISSUE TITLE
Piecing together a map

### ISSUE THREAD

**Issue by dreniarb** [overviewer/Minecraft-Overviewer#972]
If this isn't the right venue to ask this, I apologize.  I just don't know where else to look.

I'm curious what it is in the data files that tells the map how to piece each png file together?  I can extract all the relevant png's for a certain zoom level just fine, and i can piece them together like a jigsaw in photoshop, but I see no real rhyme or reason to how they're named and placed on the overall map except that a lot of them are clustered together in groups of 4 (2x2).  

Basically I'm trying to use imagemagick to create a large image each day after the map is updated.  I've poured through the files looking for some kind of database or list of files and how they fit together, but I guess I'm overlooking something.

Any help would be greatly appreciated.
